### DISCUSSION TITLE
Assess accessibility

### DISCUSSION THREAD

**Discussion by davidmyersdev** [davidmyersdev/octo#116] [General]
I want this app to be accessible. This represents assessing the current state of accessibility and creating a new milestone for it.

**Comment by vipermark7** [comment]
Something like the [WAVE](https://wave.webaim.org/extension/) browser extension or Google Chrome's Lighthouse tools could help us figure out which parts may have lower color contrast than needed, for example.

**Comment by davidmyersdev** [comment]
Awesome suggestion! Feel free (but not obligated) to test them out yourself too if you'd like. I hope to use this issue to continue conversations around what needs changed to address accessibility concerns. Otherwise, I will plan to check those tools out when I can!
